javaslang-benchmark/src/test/java/javaslang/JmhRunner.java

```diff
@@ -66,9 +66,9 @@ public static void runAndReport(Array<Class<?>> groups, int warmupIterations, in
                 .measurementTime(TimeValue.milliseconds(millis))
                 .measurementIterations(measurementIterations)
                 .forks(forks)
-                // We are using 4Gb and setting NewGen to 100% to avoid GC during testing.
-                // Any GC during testing will destroy the iteration, which should get ignored as an outlier
-                .jvmArgsAppend("-XX:+UseG1GC", "-Xss100m", "-Xms4g", "-Xmx4g", "-XX:MaxGCPauseMillis=1000", "-XX:+UnlockExperimentalVMOptions", "-XX:G1NewSizePercent=100", "-XX:G1MaxNewSizePercent=100", assertions.vmArg);
+                // We are using 2Gb and setting NewGen to 100% to avoid GC during testing.
+                // Any GC during testing will destroy the iteration (i.e. introduce unreliable noise in the measurement), which should get ignored as an outlier
+                .jvmArgsAppend("-XX:+UseG1GC", "-Xss100m", "-Xms2g", "-Xmx2g", "-XX:MaxGCPauseMillis=1000", "-XX:+UnlockExperimentalVMOptions", "-XX:G1NewSizePercent=100", "-XX:G1MaxNewSizePercent=100", assertions.vmArg);
```

Comment:
Do you know how the changes affected the benchmarks on your machine? Do the ratios change?